terraform/20-app/ecs.service.feedback-api.tf

```diff
@@ -72,14 +72,6 @@ module "ecs_service_feedback_api" {
           name      = "SECRET_KEY",
           valueFrom = aws_secretsmanager_secret.backend_cryptographic_signing_key.arn
         },
-        {
-          name      = "EMAIL_HOST_USER",
-          valueFrom = "${aws_secretsmanager_secret.private_api_email_credentials.arn}:email_host_user::"
```

Comment:
For now I don't want to change the structure of the secret itself as that would mean terraform would recreate the secret with empty values. Which would mean having to go through all relevant envs and re-populate the secret. Might be a tech debt type of job